### PR TITLE
release notes for Istio 1.8.2

### PR DESCRIPTION
**Pull request by GregHanson**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by bianpengyuan — APPROVED** [review]

File: content/en/news/releases/1.8.x/announcing-1.8.2/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.8.1 and Istio 1.8.2
+
+{{< relnote >}}
+
+## Changes
+
+- **Improved** `WorkloadEntry` auto-registration stability.
+  ([PR #29876](https://github.com/istio/istio/pull/29876))
+
+- **Improved** Multicluster install docs to include current
```
Comment (nmittler) [inline content/en/news/releases/1.8.x/announcing-1.8.2/index.md:21]: hmm ... I actually wanted this backported to 1.8.0. I guess I didn't do it right?

File: content/en/news/releases/1.8.x/announcing-1.8.2/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+best practices, incorporating recent updates to onboarding tooling. In
+particular, the multi-primary configuration (formerly known as
+"replicated control planes") no longer relies on manually configuring the
+`.global` stub domain, preferring instead to use `*.svc.cluster.local` for
+accessing services throughout the mesh.
+
+- **Deprecated** the use of the `.global` stub domain for multi-primary
```
Comment (nmittler) [inline content/en/news/releases/1.8.x/announcing-1.8.2/index.md:28]: Same here ... this isn't really new in this patch release.

File: content/en/news/releases/1.8.x/announcing-1.8.2/index.md (hunk 1 above)
Comment (GregHanson) [reply content/en/news/releases/1.8.x/announcing-1.8.2/index.md]: I moved them to the 1.8 release/upgrade notes

File: content/en/news/releases/1.8.x/announcing-1.8.2/index.md (hunk 2 above)
Comment (GregHanson) [reply content/en/news/releases/1.8.x/announcing-1.8.2/index.md]: I moved them to the 1.8 release/upgrade notes

**Review by nmittler — APPROVED** [review]

**Pull request merged**